cardhost/app/lib/builder.ts

```diff
@@ -3,38 +3,37 @@ import type {
   RawCard,
   CompiledCard,
 } from '@cardstack/core/src/interfaces';
+import { transformSync } from '@babel/core';
 import { Compiler } from '@cardstack/core/src/compiler';
-// import { compileTemplate } from 'cardhost/tests/helpers/template-compiler';
-// import templateOnlyComponent from '@ember/component/template-only';
-// import { setComponentTemplate } from '@ember/component';
-
-// import { precompile } from '@glimmer/compiler';
-// import type { TemplateFactory } from 'htmlbars-inline-precompile';
-
-// // @ts-ignore
-// import { createTemplateFactory } from '@ember/template-factory';
-// function compileTemplate(source: string): TemplateFactory {
-//   return createTemplateFactory(JSON.parse(precompile(source)));
-// }
 
+// @ts-ignore
+import HTMLBarsCompiler from 'ember-template-compiler/vender/ember-template-compiler';
```

Comment:
@ef4 Webpack doesn't like the main file of this package because it's not statically analyzable. What we ultimately want is the file under vendor (PS: Whats up with this?). Any tricks here?

Reply:
I think this is the same as `import { precompile } from '@glimmer/compiler';` that we already use in `tests/helpers/template-compiler.ts`.